app/src/main/java/org/gdg/frisbee/android/arrow/ArrowTaggedActivity.java

```diff
@@ -120,7 +118,10 @@ public void onResult(AppStateManager.StateResult stateResult) {
 
                     @Override
                     public void success(final Directory directory, final retrofit.client.Response response) {
-                        String[] orgas = taggedOrganizers.split("\\|");
+                        if (orgas != null) {
+                            return;
+                        }
+                        orgas = taggedOrganizers.split("\\|");
```

Comment:
Oh. Probably. I just left that how it was before. (Still getting to know the project better. I have read quite a bit of the overall code but not everything yet!)